tests/libtransmission/file-test.cc

```diff
@@ -659,22 +659,19 @@ TEST_F(FileTest, pathResolve)
 #ifdef _WIN32
```

Comment:
Can you please add tests for extra backslash in the middle of path, like "C:\Data\\file.txt"? For eaxmple:
![image](https://user-images.githubusercontent.com/13541699/183046238-e403a7fd-0d04-4ba8-b681-76639f1a3534.png)

Reply:
Good call. That's not new to this PR but it deserves a test anyway :+1: